client/state/data-layer/wpcom/sites/site-user-type/index.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Let's make sure to rename this file - it should map the route of the endpoint, and we renamed it to `/user-type` instead of `/site-user-type`.

Reply:
Adapted.